css/components/image.css

```diff
@@ -0,0 +1,29 @@
+.img-sm{
+    width: 200px;
+    height: 200px;
+}
+
+.img-md{
+    width: 250px;
+    height: 250px;
+}
+
+.img-lg{
+    width: 300px;
+    height: 300px;
+}
+
+.resp-img{
+    max-inline-size: 100%;
```

Comment:
you already declared width and max-inline-size in reset css.

Reply:
Oh yes, just noticed. Thanks again.
Will make the changes.